lib/OHTAPI.php

```diff
@@ -429,17 +429,17 @@ public function getContext($contextId)
      * @param integer $wordCount (optional)
      * @param string $phraseKeys (optional)
      * @param boolean $retranslate (optional)
-     * @param string $callbackUrl (optional)
+     * @param string $notes
      * @return stdClass response object
      */
-    public function translateContext($contextId, $source, $target, $wordCount = 0, $phraseKeys = '', $retranslate = false, $callbackUrl = '')
+    public function translateContext($contextId, $source, $target, $wordCount = 0, $phraseKeys = '', $retranslate = false, $notes = '')
```

Comment:
@oyagev thanks for the info. I was using an outdated version which didn't have the callback. When I pasted it in I didn't notice the change in the diff. Thanks for pointing it out. I'll update now.